hamster_gtk/preferences/widgets/hamster_spin_button.py

```diff
@@ -43,31 +43,36 @@ class HamsterSpinButton(Gtk.SpinButton, ConfigWidget):
     # Required else you would need to specify the full module name in ui file
     __gtype_name__ = 'HamsterSpinButton'
 
-    def __init__(self, adj=None):
+    def __init__(self, adj=None, climb_rate=0, digits=0):
         """
         Initialize widget.
 
         Args:
-            adj (Gtk.Adjustment, SimpleAdjustment): Adjustment for the widget, either
+            adj (Gtk.Adjustment, SimpleAdjustment, optional): Adjustment for the widget, either
                 :class:`Gtk.Adjustment` or a :class:`SimpleAdjustment`.
-                See their respective documentation for more information.
+                See their respective documentation for more information. Defaults to ``None`` in
+                which case it can be set later.
         """
-        super(Gtk.Entry, self).__init__()
+        super(Gtk.SpinButton, self).__init__(climb_rate=climb_rate, digits=digits)
```

Comment:
Yes, all the PyGI constructors do for some reason. They do not seem to be used though.